app/views/admin/products_v3/_filters.html.haml

```diff
@@ -4,10 +4,10 @@
       = text_field_tag :search_term, search_term, placeholder: t('.search_products')
   .producers
     .label= t('.producers.label')
-    = select_tag :producer_id, options_for_select(producer_options, producer_id), include_blank: t('.all_producers')
+    = select_tag :producer_id, options_for_select(producer_options, producer_id), include_blank: t('.all_producers'), "data-controller": "tom-select", "data-tom-select-options-value": '{ "plugins": [] }', class: "fullwidth"
```

Comment:
Why is it not the right usage of Tom Select? 

I see that you're removing the plugin of "dropdown_input", which makes sense to me. 
But ongoing, perhaps we can create a simpler config option, with some pre-defined types. Eg:

`data-tom-select-type="default"` (a plain simple dropdown)
`data-tom-select-type="searchable"` (has an input that you can type into)
... and a multi-select, etc..

Then in the stimulus controller we configure it as needed.

Reply:
Co-incidentally, Mario has mentioned the different types at the same time here: https://openfoodnetwork.slack.com/archives/C01CXQNJ1J6/p1690330195194939?thread_ts=1690181990.070429&cid=C01CXQNJ1J6

Reply:
> perhaps we can create a simpler config option, with some pre-defined types

Yes, it's probably a good idea, instead of using that `options-value` which is a bit tricky to use.

Reply:
Actually, we are not ready to use this: dropdowns and their usage/design are not well fixed. I'd prefer to postpone this once it get determined.

Reply:
Yes, good point 👍